GraphQL: Add File & LogInWith (#702)
ParsePlatform/Docs: _includes/graphql/files.md

The GraphQL API supports file upload via [GraphQL Upload](https://github.com/jaydenseric/graphql-upload), to send a `File` through `GraphQL` it's recommended to use the [Apollo Upload Client](https://github.com/jaydenseric/apollo-upload-client).

> ## Add a File field to Class

Reviewer: ```suggestion
## Add a File field to a Class
```

}
```

> ## Create File

Reviewer: ```suggestion
## Create a File
```

> GraphQL API support nested mutation for the `File` type, so you can send the file along the Parse Object or just upload the file and get returned information.

Reviewer: ```suggestion
The GraphQL API supports nested mutation for the `File` type, so you can send the file along with the Parse Object or just upload the file and get the returned information.
```

  }
}
```

## Create and add a file

> Lets create a new `GameScore` object and upload the file along the creation.

Reviewer: ```suggestion
Lets create a new `GameScore` object and upload the file.
```
